integration/generator/test_case_generator.go

```diff
@@ -20,12 +20,17 @@ const (
 
 //you can't have a const map in golang
 var osToTestDirMap = map[string][]string{
-	linux: {"./integration/test/ca_bundle",
+	"ec2_linux": {
+		"./integration/test/ca_bundle",
 		"./integration/test/cloudwatchlogs",
-		"./integration/test/metrics_number_dimension"},
+		"./integration/test/metrics_number_dimension",
+	},
 	// @TODO add real tests
-	windows: {""},
-	mac:     {},
+	"ec2_windows": {""},
+	"ec2_mac":     {},
+	"ecs_fargate": {
```

Comment:
We should try to run this from the root dir so we can use ./

Reply:
I would agree at some point. However, it would pose a different question? Would the developers prefer to look at the root dir or from the current path since for ECS Fargate test, it use 
```
resource "null_resource" "validator" {
  provisioner "local-exec" {
    command = <<-EOT
    EOT
  }
}
```
instead of like EC2
```
resource "aws_instance" "integration-test" {
  provisioner "remote-exec" {
}
```
Therefore, would prefer this path over root_dir. One more addition point to support mine, since I use reduce the variables by using [local](https://github.com/aws/amazon-cloudwatch-agent/pull/458/files#diff-5cf3a68c413f5f031a2713f7488ab0c4668f7a4895120e1dfba706c50b28bba6R18-R23), so it would be also a pain for using the root dir if don't do something to the root dir.

Reply:
If you guys prefer root module, then its fine by me also.